.github/workflows/daily.yaml

```diff
@@ -199,13 +199,14 @@ jobs:
         shell: bash
         run: tar -xzf maven-repo.tgz -C ~
       - name: Install cl.exe
-        uses: ilammy/msvc-dev-cmd@v1
```

Comment:
Please see https://github.com/ilammy/msvc-dev-cmd/tags, both ilammy/msvc-dev-cmd@v1 and ilammy/msvc-dev-cmd@v1.12.0 are tags for commit [7315a94](https://github.com/ilammy/msvc-dev-cmd/commit/7315a94840631165970262a99c72cfb48a65d25d). Why is this change needed?

Reply:
You say we should switch from https://github.com/ilammy/msvc-dev-cmd/commit/7315a94840631165970262a99c72cfb48a65d25d to https://github.com/ilammy/msvc-dev-cmd/commit/7315a94840631165970262a99c72cfb48a65d25d because v1 is same as v.12.0

Reply:
sorry, not sure if I get you... 

I'm just trying to update that GHA from `v1` to `v1.12.0` as latter is the one updated to use Node16

Reply:
thank you @michalvavrik, didn't spot it

Reply:
done